components/contribution-flow/index.js

```diff
@@ -872,6 +910,55 @@ class CreateOrderPage extends React.Component {
               customData={customData}
               onCustomFieldsChange={this.handleCustomFieldsChange}
             />
+            <FeesOnTopContainer p={3} mt={3}>
+              <Box maxWidth="75%">
+                <P fontSize="Caption" my={2}>
+                  <FormattedMessage
+                    defaultMessage="Open Collective has waived 100% of our fees for COVID-19 Relief Collectives. We rely on the generosity
+                of contributors like you to keep offering this to Collectives."
+                    id="feesOnTop.info"
+                  />
```

Comment:
```suggestion
                  <FormattedMessage
                    defaultMessage="Open Collective has waived 100% of our fees for COVID-19 Relief Collectives. We rely on the generosity of contributors like you to keep offering this to Collectives."
                    id="feesOnTop.info"
                  />
```

Reply:
Oh, that's a weird formatting. I wonder why it did that 🤔